luigi/contrib/kubernetes.py

```diff
@@ -217,6 +302,7 @@ def run(self):
                 }
             },
             "spec": {
+                "activeDeadlineSeconds": 100,
```

Comment:
If I get this correctly, this should be passed as parameter and be way higher IMO, I have jobs that take way longer to complete.

Reply:
So I think this name (& the [k8s doc](https://kubernetes.io/docs/concepts/workloads/controllers/jobs-run-to-completion/#job-termination-and-cleanup)) is confusing: It seems to me that it's the deadline for the Job to be scheduled. Once the Job is running, this timeout doesn't apply anymore.

Reply:
So does this apply to `ContainerCreating`? If so I would remove this line just to be safe, or at least I would have it parametrized.

Reply:
I don't understand the safety concern with this parameter: my last commit `don't fail on ContainerCreating` means that `ContainerCreating` (which happens to me if I push a new docker image and kickoff luigi right after) is not considered as an error and allows retries. 
`activeDeadlineSeconds ` mean that if the job retries for 100s and still doesn't get scheduled, the luigi task should fail (instead of potentially hang for ever).
Agreed about the parameterization.